docs/pages/Specs/nexus-prover.mdx

```diff
@@ -35,32 +35,124 @@ After receiving a Merkle proof from the initialization step with a Merkle proof
+
+For IVC proofs associated with the secondary `grumpkin` curve, the same combination used for the primary curve is not possible as `grumpkin` is not a pairing-friendly elliptic curve. As a result, the current design of the Nexus zkVM uses a variant of [Halo2](https://zcash.github.io/halo2/) by the Ethereum's Privacy-Scaling-Explorations team based on the Kate-Zaverucha-Goldberg (KZG) polynomial commitment scheme [[KZG10](#references)]. Though this part of the proof system is required to perform in-circuit multiscalar multiplications, the blowup in the circuit size is manageable due to the smaller size of the secondary IVC circuit. We remark, however, that this part of the design is currently under active development and may change.
+
+To implement the second part of the compression phase, the current plan is for the Nexus zkVM to use Groth16 proofs [[Gro16](#references)]. This final stage of compression will recursively verify the Spartan proofs from the first stage, reducing the log-sized proofs to a constant size.
+
+## Nexus zkVM parameters
+
+In order to run the Nexus zkVM, certain values used by the proof system as well as the public parameters shared between the prover and the verifier need to be configured.
+
+Two of the main configuration parameters for the Nexus zkVM are:
+- Configuration parameters for the Poseidon hash function; and
+- The number $k$ of Nexus VM instructions per folding step.
+
+As for the public parameters, these consist essentially of the following:
+- Parameters for the Pedersen commitment used by Nova in the proof accumulation phase for each curve in the cycle; 
+- Parameters for the Zeromorph commitment used by Spartan in the compression phase; and
+- Parameters for the remaining SNARKs in the compression phase.
 
+### Hash function configuration 
+
+In order to configure the Poseidon hash function, we currently use the parameters proposed in the Poseidon paper [[GKR21](#references)] to achieve a 128-bit level of security. More precisely, we use the following parameters:
+
+```rust
+const FULL_ROUNDS: usize = 8;
+const PARTIAL_ROUNDS: usize = 57;
+const ALPHA: u64 = 5;
+const RATE: usize = 2;
+const CAPACITY: usize = 1;
+const MODULUS_BITS: u64 = 254;
+```
+
+### Instructions per folding step
+
+The Nexus zkVM's approach based on Nova provides a proof system with a recursion overhead which is independent of the total number of computation steps and of the step function itself. However, in our current implementation, the recursion overhead  remains quite high (in the order of $70{,}000$ and $340{,}000$ constraints in the IVC and PCD settings) compared the cost of a single step of the Nexus VM (around $15{,}000$ constraints). 
```

Comment:
same comment on the IVC overhead

Reply:
I'm updating the value of this he IVC overhead in this he text and adjusting the value of k in the example.